### ISSUE TITLE
Ability to pass `run.experiment.name` and `run_hash` to the pl adapter

### ISSUE THREAD

**Issue by constd** [aimhubio/aim#2581]
## 🚀 Feature

Add the ability pass a run name (in case of a new run) or a run hash (in case of an existing run) to the pytorch lightning logger.

### Motivation

Currently, using the AimLogger adapter for pytorch lightning makes it impossible to pass in a name for a run, resulting in a lot of runs names like `Run_<hash>` which make it hard to follow/track basic differences. At the same time, resuming a run, requires passing in a `run_hash` immediately after instantiation (and before `self.experiment` is accessed).

### Pitch

I would like to pass in run `name` and `hash` to the pytorch lightning aim logger, adding support for custom names and resuming training runs.

### Alternatives

```
logger = AimLogger()
logger._run_hash = <hash_number>
```
and
```
logger = AimLogger()
logger.experiment.name = <run_name>
```

**Comment by alberttorosyan** [comment]
Hey @constd! Thanks for submitting the issue.
Putting the discord convo [link](https://discord.com/channels/1047782741736968202/1083237141531856937) for future reference.
